x/ecocredit/server/core/features/msg_update_project_admin.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Msg/UpdateProjectAdmin
+
+  The admin of a project can be updated:
+  - when the project exists
+  - when the admin is the admin of the project
+  - the project admin is updated
+
+  Rule: The project must exist
+
+    Background:
+      Given a credit type with abbreviation "C"
+      And a credit class with class id "C01"
+      And a project with project id "C01-001" and admin alice
+
+    Scenario: the project exists
+      When alice attempts to update project admin with project id "C01-001"
+      Then expect no error
+
+    Scenario: the project does not exist
+      When alice attempts to update project admin with project id "C01-002"
+      Then expect the error "could not get project with id C01-002: not found: invalid request"
+
+  Rule: The admin must be the admin of the project
```

Comment:
```suggestion
  Rule: The user must be the admin of the project
```
i think this was referring to the field `admin` but i think `user` would be more natural here

Reply:
Good point. I'm leaning towards using the field name, hence the change here, but I also see value in using "user". In some cases the address might not be a "user" but rather a "group" or a multisig and therefore "signer" might make sense too. I'm going to leave this here now but we should discuss when writing guidelines so I've left a comment in the issue. #1242

Reply:
I'm leaning towards using the field name, hence the change here, but I also see value in using "user". In some cases the address might not be a "user" but rather a "group" or a multisig and therefore "signer" might make sense too. I'm going to leave this here now but we should discuss when writing guidelines so I've left a comment in the issue. #1242